e2e/utils/card.js

```diff
@@ -0,0 +1,25 @@
+const { scrollBy } = require('./scroll');
+
+const isCardExisting = async(browser, cardSelector) => {
+	const card = await browser.$(cardSelector);
+
+	let isExisting = false;
+
+	await browser.waitUntil(async()=>{
+		isExisting = await card.isExisting();
+
+		await scrollBy(browser, 20);
+
+		return isExisting;
+	}, {
+		timeout: 10000,
+		interval: 1000,
```

Comment:
Could this interval be one of the reasons why are we slow?

Reply:
That the card exists is critical for all the tests, so we have to wait the time needed for it to appear. In most cases it appear immediately.

Reply:
> That the card exists is critical for all the tests, so we have to wait the time needed for it to appear. In most cases it appear immediately.

I was thinking more in terms if 1000 is too much or not